tests/test_weighted_pandas.py

```diff
@@ -1,9 +1,11 @@
 from anesthetic.weighted_pandas import WeightedDataFrame, WeightedSeries
+from anesthetic.utils import channel_capacity
 from pandas import Series, DataFrame
 import pytest
 import numpy as np
 from numpy.testing import assert_array_equal, assert_allclose
 import matplotlib.pyplot as plt
+from pandas.plotting import scatter_matrix, bootstrap_plot
```

Comment:
Shouldn't this be `from anesthetic.plotting import scatter_matrix, bootstrap_plot`?

Reply:
You can see something of the intricacy of the backend system here. `pandas.plotting.scatter_matrix` calls the backend. Originally this would then have directed to `pandas.plotting._matplotlib.misc.scatter_plot`, but now this directs to `anesthetic.plotting._matplotlib.misc.scatter_plot`. It's important that it does this, since this way if you call native pandas functions on weighted data frames/series it still uses the weights, whilst still doing the unweighted job when there are no weights.

Reply:
Ouf, that's subtle, nice. 

Are we testing for this, i.e. are we testing that `scatter_matrix` and `bootstrap_plot` still behave as usual for **un**weighted samples?

If not, is there an easy way of doing that?

Reply:
Good point -- I have added a test which checks that the `scatter_matrix` call is indeed different from the `pandas.plotting._matplotlib.misc.scatter_matrix` call.